Header header

int8 sensor1
int8 sensor2
int8 sensor4
int8 sensor5